aws/resource_aws_acm_certificate_validation.go

```diff
@@ -101,9 +113,16 @@ func resourceAwsAcmCertificateCheckValidationRecords(validationRecordFqdns []int
 			cert = output.Certificate
 			return nil
 		})
+		if isResourceTimeoutError(err) {
+			output, err = conn.DescribeCertificate(input)
+			if len(output.Certificate.DomainValidationOptions) == 0 {
```

Comment:
To prevent potential panics here, we should either perform an `err` check before accessing `output` or perform `nil` checks, e.g.

```suggestion
			if output != nil && output.Certificate != nil && len(output.Certificate.DomainValidationOptions) == 0 {
```